src/app/core/services/http-ingest.service.ts

```diff
@@ -0,0 +1,21 @@
+import { environment } from '../../../environments/environment'
+import { Injectable } from '@angular/core'
+import { HttpClient } from '@angular/common/http'
+import { HttpErrorService } from './http-error.service'
+import { IngestService } from '../abstractions/ingest-service'
+import { catchError, Observable } from 'rxjs'
+
+const INGEST_URL: string = `${environment.apiBaseUrl}/ingest-operations`
+
+@Injectable()
+export class HttpIngestService implements IngestService {
+  constructor(
+    private readonly http: HttpClient,
+    private readonly httpErrorService: HttpErrorService
+  ) {}
+
+  public reingestRundownData(rundownId: string): Observable<void> {
```

Comment:
I don't think the concept of ingest is necessary for this feature as mentioned in Sofie server. This is still an action on the rundown, and can be moved to RundownService (HttpRundownService).

Reply:
The concept has been removed and the operation is on the RundownService instead.